pyfr/plugins/base.py

```diff
@@ -59,6 +59,9 @@ def __init__(self, intg, cfgsect, suffix=None):
     def __call__(self, intg):
         pass
 
+    def get_data(self):
```

Comment:
Maybe something like serialise is a better name?  Also, is there any benefit to passing intg?